pkg/csi/service/wcpguest/controller.go

```diff
@@ -332,12 +332,16 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 	resp, faultType, err := createVolumeInternal()
 	log := logger.GetLogger(ctx)
 	log.Debugf("createVolumeInternal: returns fault %q", faultType)
+
+	namespace := common.GetNamespaceFromContext(ctx)
```

Comment:
Are we planning to customize the side cars for PVCSI as well?

Reply:
It can be used if we plan to customize them in future. For now, I can directly put the namespace as Unknown as done for labels in vanilla.

Reply:
I thought we do not want to customize the side cars for PVCSI. And the plan is to do that only for WCP.

Reply:
Namespace set as unknown for PVCSI operations.